src/dev/src/app/radios/radios.demo.html

```diff
@@ -1,65 +1,279 @@
+<!--<span class="clr-subtext">Helper Text</span>-->
+<!--</div>-->
+<!--</div>-->
+
+<!--<div class="clr-form-control">-->
+<!--<label class="clr-control-label">-->
+<!--Inline radio</label>-->
+<!--<div class="clr-control-container">-->
+<!--<div class="clr-radio-wrapper clr-radio-inline">-->
+<!--<input type="radio" name="radio3" id="compact-radio7" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio7" class="clr-control-label">option 1</label>-->
+<!--<input type="radio" name="radio3" id="compact-radio8" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio8" class="clr-control-label">option 2</label>-->
+<!--<input type="radio" name="radio3" id="compact-radio9" placeholder="Enter value here" class="clr-radio">-->
+<!--<label for="compact-radio9" class="clr-control-label">option 3</label>-->
+<!--</div>-->
+<!--&lt;!&ndash; IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER &ndash;&gt;-->
+<!--<clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>-->
+<!--<span class="clr-subtext">Helper Text</span>-->
+<!--</div>-->
+<!--</div>-->
+<!--</form>-->
+
+<h2>Angular Radio</h2>
+
+<h4>Vertical</h4>
+
+<label>One <input type="radio" name="one" value="one" [(ngModel)]="vertical.one" /></label>
+<label>Two <input type="radio" name="one" value="two" [(ngModel)]="vertical.one" /></label>
+<label>Three <input type="radio" name="one" value="three" [(ngModel)]="vertical.one" /></label>
+
+<!--<p>Selected: {{vertical.one}}</p>-->
+
+<form clrForm>
+  <!--<clr-radio-container>-->
+    <!--<label>First</label>-->
+    <!--<input id="vertical-one" clrRadio [(ngModel)]="vertical.one" name="one" required />-->
+    <!--<label for="vertical-one">Radio Label</label>-->
+  <!--</clr-radio-container>-->
+
+  <clr-radio-container>
+    <label>Second</label>
+    <clr-radio-wrapper>
+      <label>One</label>
+      <!-- type="radio" required for angular to pick up this input correctly -->
+      <input clrRadio type="radio" [(ngModel)]="vertical.two" value="one" name="two" required />
+    </clr-radio-wrapper>
+    <clr-radio-wrapper>
+      <label>Two</label>
+      <input clrRadio type="radio" [(ngModel)]="vertical.two" value="two" name="two" required />
+    </clr-radio-wrapper>
+    <clr-radio-wrapper>
+      <label>Three</label>
+      <input clrRadio type="radio" [(ngModel)]="vertical.two" value="three" name="two" required />
+    </clr-radio-wrapper>
+    <clr-control-helper>Helper text</clr-control-helper>
+    <clr-control-error>There was an error</clr-control-error>
+  </clr-radio-container>
+
+  <p>Selected: {{vertical.two}}</p>
+
+  <clr-radio-container>
+    <label>Third</label>
+    <clr-radio-wrapper>
+      <!--<label>One</label>-->
+      <!-- type="radio" required for angular to pick up this input correctly -->
+      <input clrRadio type="radio" [(ngModel)]="vertical.three" value="one" name="three" required />
```

Comment:
This use case doesn't exist. You can't put 3 radio buttons without labels together, it makes no sense for the user.

Reply:
Nope, again trial and error failure.